README.md

```diff
@@ -53,3 +53,7 @@ Every SynergyCP product on WHMCS must include the following Configurable Options
 | Drive Bay 1, Drive Bay 2, etc. (any number of entries) | 1TB-HDD&#124;1 TB Hard Drive                              | The billing ID value should be None for empty disk bays.                                                     |
 | Operating System                                       | os-centos6&#124;CentOS 6                                  | The billing ID must match an OS Reload Profile's Billing ID on SynergyCP                                     |
 | Bandwidth                                              | 20000GB&#124;20 TB Bandwidth or &#124;Unmetered Bandwidth | Optional. The billing ID **MUST** use the same unit as your overage billing setting in WHMCS (default is GB) |
+
+**Preset Configurable Options**:
+
+Under Products/Services < Module Settings there are the MEM Billing ID, Addon Billing IDs, and Disk Billing IDs fields. These can be set as default preset values for these fields. If the corresponding Configurable Option field is included it will override its respective field. These options can be used for setups with complete or partial pre-configuration of servers. The fields use a comma-separated format that allows for multiple values: add-1, add-2|Addon1, Addon2
```

Comment:
couple very minor nits:

```suggestion
Under Products/Services > Module Settings there are the Memory Billing ID, Addon Billing IDs, and Disk Billing IDs fields. These can be set as default preset values for these fields. If the corresponding Configurable Option field is included it will override its respective field. These options can be used for setups with complete or partial pre-configuration of servers. The fields use a comma-separated format that allows for multiple values: add-1, add-2|Addon1, Addon2
```